README.md

```diff
@@ -144,6 +164,82 @@ Documentation for the flow package can be accessed using standard godoc tool, e.
 godoc github.com/trustmaster/goflow
 ```
 
+## Runtime registration example
```

Comment:
If runtime registration is method "Runtime.Register()," would it be called on an as-needed basis by the user using a console argument? e.g. ">> goflow -Register," or something?  Otherwise the user is going to end up with a billion copies of registered runtimes.